ConsentViewController/Classes/SourcePointClient/Adapters.swift

```diff
@@ -255,6 +257,48 @@ extension SPMobileCore.USNatConsent {
     }
 }
 
+extension SPMobileCore.PreferencesConsent.PreferencesSubType? {
+    func toNative() -> SPPreferencesConsent.PreferencesSubType {
+        switch self {
+        case .aipolicy: return .AIPolicy
+        case .legalpolicy: return .LegalPolicy
+        case .privacypolicy: return .PrivacyPolicy
+        case .termsandconditions: return .TermsAndConditions
+        case .termsofsale: return .TermsOfSale
+        case .unknown: return .Unknown
+        default: return .Unknown
+        }
+    }
+}
+
+extension SPMobileCore.PreferencesConsent.PreferencesStatusPreferencesChannels {
+    func toNative() -> SPPreferencesConsent.PreferencesChannels { return .init(channelId: Int(self.channelId), status: self.status) }
```

Comment:
when a function has a single expression, the `return` keyword can be omitted. 

Could you remove it?

Reply:
I was referencing https://github.com/SourcePointUSA/android-cmp-app/pull/859 PR